### ISSUE TITLE
Semigroups package does not currently compile in Travis

### ISSUE THREAD

**Issue by wilfwilson** [gap-packages/Semigroups#255]
And therefore the tests are failing.

The error can be viewed here, for example: [https://travis-ci.org/gap-packages/Semigroups/jobs/201076253](https://travis-ci.org/gap-packages/Semigroups/jobs/201076253)
```
config.status: executing libtool commands
  CXX    src/semigroups_la-gap.lo
  CXX    src/semigroups_la-converter.lo
  CXX    src/semigroups_la-bipart.lo
  CXX    src/semigroups_la-ufdata.lo
  CXX    src/semigroups_la-fropin.lo
  CXX    src/semigroups_la-congpairs.lo
  CXX    src/semigroups_la-semifp.lo
  CXX    src/semigroups_la-semigrp.lo
  CXX    src/libsemigroups/semigroups_la-blocks.lo
  CXX    src/libsemigroups/semigroups_la-cong.lo
  CXX    src/libsemigroups/cong/semigroups_la-kbfp.lo
  CXX    src/libsemigroups/cong/semigroups_la-kbp.lo
src/libsemigroups/cong/kbp.cc: In member function ‘virtual void libsemigroups::Congruence::KBP::run()’:
src/libsemigroups/cong/kbp.cc:45:30: error: ‘really_delete_cont’ was not declared in this scope
       really_delete_cont(gens);
                              ^
make: *** [src/libsemigroups/cong/semigroups_la-kbp.lo] Error 1
```

**Comment by james-d-mitchell** [comment]
Ah I forgot to change some stuff in Semigroups after some changes in
Libsemigtoups

**Comment by wilfwilson** [comment]
This is resolved by commit 89796d9
